kotlinx-support-jdk7/src/main/kotlin/Paths.kt

```diff
@@ -0,0 +1,157 @@
+@file:Suppress("NOTHING_TO_INLINE")
+package kotlinx.support.jdk7
+
+import java.io.BufferedReader
+import java.io.BufferedWriter
+import java.io.InputStream
+import java.io.OutputStream
+import java.nio.charset.Charset
+import java.nio.charset.StandardCharsets
+import java.nio.file.*
+import java.nio.file.attribute.FileTime
+import java.nio.file.attribute.UserPrincipal
+import java.util.*
+import java.util.stream.Collectors
+
+/** Converts a path string to a [Path]. */
+inline fun String.toPath() = Paths.get(this)
```

Comment:
Consider also `pathOf(String): Path` and pathOf(vararg String): Path` functions.